src/docfx/build/legacy/LegacyFileMap.cs

```diff
@@ -61,7 +61,7 @@ public static void Convert(Docset docset, Context context, IEnumerable<(string p
                 {
                     host = docset.HostName,
                     locale = docset.Locale,
-                    base_path = $"/{docset.SiteBasePath}",
+                    base_path = $"/{PathUtility.NormalizeSiteBasePath(docset.SiteBasePath)}",
```

Comment:
Instead of normalize the site base path on each consume, it is better to normalize it in `SpliteBaseUrl` method, the `siteBasePath` property produced by that method should return a normalized URL that always starts with `/`

Reply:
This and #5176 are caused by the same problem and can be fixed using the aforementioned method